File: Code/cpp/pattern3.cpp
#include<iostream>
using namespace std;
int main()
{
    int limit, i, j;

    cout << "Enter Limit: ";
    cin >> limit;

    for (i = 1; i < limit; i++)
    {
        for (j = 0; j < i; j++)
            cout << "* ";
        cout << "\n";
    }
    for (i = limit; i > 0; i--)
    {
        for (j = i; j > 0; j--)
            cout << "* ";
        cout << "\n";
    }
    return 0;
}

File: Code/java/pattern3.java
import java.util.Scanner;

class Pattern3
{
    public static void main(String[] args)
    {
        int limit, i;
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Limit: ");
        limit = scanner.nextInt();

        for (i = 1; i < limit; i++)
            System.out.println("* ".repeat(i));
        for (i = limit; i > 0; i--)
            System.out.println("* ".repeat(i));

        scanner.close();
    }
}